adapters/Mintegral/MintegralAdapter/GADMediationAdapterMintegralConstants.h

```diff
@@ -15,7 +15,7 @@
 #import <Foundation/Foundation.h>
 
 /// Mintegral mediation adapter version.
-static NSString *const GADMAdapterMintegralVersion = @"7.1.9.0.0";
+static NSString *const GADMAdapterMintegralVersion = @"7.1.9.0";
```

Comment:
Revert, the SDK version in CocaoPods is `7.1.9.0`, so the adapter version would be `7.1.9.0.0`.